examples/seq2seq/test_finetune_trainer.py

```diff
@@ -1,181 +0,0 @@
-# Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
You can't just remove this file - almost all of these are important core tests. They need to be ported to the new way.

Reply:
e.g. fairscale isn't being tested anywhere else, same with apex.

Reply:
these tests are here because they need a functional program to be tested on and not a mock-up, I keep saying that there is conflict between examples and how they are being used. But I'm not getting anywhere with this.

Reply:
I probably need to port deepspeed tests as well, since this PR will break all of them.

Reply:
It's fine by me if we need to keep these tests, I decided to remove these since all other legacy examples don't have tests.

Think we should add the fairscale test to `/examples` with the new script instead of `legacy` as it's an important test.

Reply:
That's is what I'm trying to say - these tests are not examples, but core functionality and ideally should be under `tests`, but they need real programs to be tested on. So this is a grey zone.

Reply:
Do you think you could re-write those tests with the new script? And merge this after that.

Reply:
Yes, I will do it.